news/10647.feature.rst

```diff
@@ -0,0 +1,9 @@
+Allow Python distributors to opt-out from or opt-in to the ``sysconfig``
```

Comment:
Can we shorten this news fragment, to a single sentence?

I think the target audience for this change would end up looking up the issue/PR anyway, and most end users will not care about this.

Reply:
Will do.

Reply:
I've editted OP to keep the original changelog entry, which should aid discoverability.